netbox/models/writable_ip_address.go

```diff
@@ -107,7 +107,7 @@ type WritableIPAddress struct {
 	Status string `json:"status,omitempty"`
 
 	// tags
-	Tags []*NestedTag `json:"tags"`
+	Tags []*NestedTag `json:"tags,omitempty"`
```

Comment:
You probably generated this change from the swaggerfile. I changed this manually in the code, so your re-generation changed it back. I need this change in the terraform provider and I dont think you meant to commit it, since it does not correspond to the LocalContextData stuff. Please discard this change.